sklearn/covariance/graph_lasso_.py

```diff
@@ -557,7 +557,7 @@ def fit(self, X, y=None):
         X : ndarray, shape (n_samples, n_features)
             Data from which to compute the covariance estimate
         """
-        X = check_array(X)
+        X = check_array(X, ensure_min_features=2)
```

Comment:
This PR also makes sure that sensible errors are raised.
GraphLasso creates a covariance matrix, which doesn't make sense for 1 feature.
That is probably what the comment should say.